baseapp-email-templates/setup.py

```diff
@@ -1,12 +1,18 @@
-from setuptools import find_packages, setup
+from setuptools import setup
+
+VERSION = "1.3"
+
 
 with open("README.md", "r") as f:
     long_description = f.read()
 
+
 setup(
     name="baseapp-email-templates",
-    packages=find_packages(),
-    version="0.1",
+    description="baseapp-email-templates is now baseapp-message-templates",
     long_description=long_description,
     long_description_content_type="text/markdown",
+    version=VERSION,
+    install_requires=["baseapp-message-templates"],
```

Comment:
the package don't need to require itself, we can remove `install_requires`

Reply:
I think we do want to require this since we want the old package to point to this new package if somene runs `pip install baseapp-email-templates`.

I took this change from this steps https://github.com/simonw/pypi-rename